backend/src/apiserver/client_manager/client_manager.go

```diff
@@ -506,3 +584,12 @@ func backfillExperimentIDToRunTable(db *gorm.DB) error {
 	`)
 	return err
 }
+
+// Returns the same error, if it's not "already exists" related.
+// Otherwise, return nil.
+func ignoreAlreadyExistError(err error) error {
+	if err != nil && strings.Contains(err.Error(), "exists") {
```

Comment:
This method seems very fragile. I imagine there's could be multiple scenarios matching keyword "exists".
By the way, why do we need to ignore this now? Was it an issue for MySQL in the past or does only applies to Postgres?

Reply:
I added this because MySQL simply ignores if the foreign key already exist, but PostgreSQL reports an error. A more robust way I can think of is to use an if/else branch for the two SQLs and handle them separately.